colour/difference/delta_e.py

```diff
@@ -560,3 +566,56 @@ def delta_E_ITP(ICtCp_1: ArrayLike, ICtCp_2: ArrayLike) -> NDArrayFloat:
     )
 
     return as_float(d_E_ITP)
+
+
+def delta_E_HyAB(Lab_1: ArrayLike, Lab_2: ArrayLike) -> NDArrayFloat:
+    """
+    Return the difference between two *CIE L\\*a\\*b\\** colourspace arrays
+    This metric is intended for large color differences,
+    on the order of 10 CIELAB units or greater
+
+    Parameters
+    ----------
+    Lab_1
+        *CIE L\\*a\\*b\\** colourspace array 1.
+    Lab_2
+        *CIE L\\*a\\*b\\** colourspace array 2.
+
+    Returns
+    -------
+    :class:`numpy.ndarray`
+        Colour difference HyAB.
+
+    Notes
+    -----
+    +------------+-----------------------+-------------------+
+    | **Domain** | **Scale - Reference** | **Scale - 1**     |
+    +============+=======================+===================+
+    | ``Lab_1``  | ``L_1`` : [0, 100]    | ``L_1`` : [0, 1]  |
+    |            |                       |                   |
+    |            | ``a_1`` : [-100, 100] | ``a_1`` : [-1, 1] |
+    |            |                       |                   |
+    |            | ``b_1`` : [-100, 100] | ``b_1`` : [-1, 1] |
+    +------------+-----------------------+-------------------+
+    | ``Lab_2``  | ``L_2`` : [0, 100]    | ``L_2`` : [0, 1]  |
+    |            |                       |                   |
+    |            | ``a_2`` : [-100, 100] | ``a_2`` : [-1, 1] |
+    |            |                       |                   |
+    |            | ``b_2`` : [-100, 100] | ``b_2`` : [-1, 1] |
+    +------------+-----------------------+-------------------+
+
+    References
+    ----------
+    :cite:`Abasi2020`
+
+    Examples
+    --------
+    >>> Lab_1 = np.array([25.0, 8.0, -14.0])
+    >>> Lab_2 = np.array([75.0, -35.0, 16.0])
+    >>> delta_E_CIE2000(Lab_1, Lab_2)  # doctest: +ELLIPSIS
```

Comment:
**Major**

The unit tests are likely failing because of this one, it should be calling `delta_E_HyAB` and not `delta_E_CIE200`.

Reply:
Thanks!